book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Well done initializing the correct parameters for the Book class. However, for the Rentals to be created correctly, it requires a custom method called `add_rentals` or whatever you may name it as shown below.
```ruby
  def add_rental(person, date) 
    Rental.new(date, self, person)
  end
  ```